packages/core/content-manager/server/routes/admin.js

```diff
@@ -84,19 +84,15 @@ module.exports = {
       path: '/relations/:model/:targetField',
       handler: 'relations.findAvailable',
       config: {
-        policies: [
-          'admin::isAuthenticatedAdmin',
-          {
-            name: 'plugin::content-manager.hasPermissions',
-            config: {
-              actions: [
-                'plugin::content-manager.explorer.create',
-                'plugin::content-manager.explorer.update',
-              ],
-              hasAtLeastOne: true,
-            },
-          },
-        ],
+        policies: ['admin::isAuthenticatedAdmin'],
+      },
+    },
+    {
+      method: 'GET',
+      path: '/relations/:model/:id/:targetField',
+      handler: 'relations.findExisting',
+      config: {
+        policies: ['admin::isAuthenticatedAdmin'],
```

Comment:
How will we syncronize with frontend to merge this? As it breaks the content manager with his change.

We will merge https://github.com/strapi/strapi/pull/14544 with this one at the same time?

Reply:
yes :)